buddyboss/buddyboss-platform
# PROD-909 Disable the registration page mapping on activity settings save

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1553 .

### How to test the changes in this Pull Request:

1. Enable Registration from general settings
2. Move on the page settings and unset the Register Form page from the registration pages section
3. Move on the activity settings page and save it
4. Back on page settings and check the Register Form page option
5. The option will not auto-assign on the last save action

### Proof Screenshots or Video
https://www.awesomescreenshot.com/video/5971551?key=4d3ef947971fe6b038d3f80c9c200e3a

### Other information:




### Changelog entry

> Disable the registration page mapping on activity settings save action
